dashboard/templates/metric_selections_phantom_snip.html

```diff
@@ -26,13 +26,14 @@ <h4>
 
 <div class="btn-group col-md-3" role="group" aria-label="scan tags" id="scantypeselector">Select scan subtype:</br>
   {% for scantype in study.scantypes %}
-    {% if scantype.name in ['T1'] %}
-      <label style="display:none" class="scantype_t1"> {{ scantype.name }}<input type="radio" aria-label="{{scantype.name}}" value="{{scantype.name}}" name="scantype" style="margin-top: 5px;"/></label>
-    {% elif scantype.name.startswith('DTI') %}
-      <label style="display:none" class="scantype_dti"> {{ scantype.name }}<input type="radio" aria-label="{{scantype.name}}" value="{{scantype.name}}" name="scantype" style="margin-top: 5px;"/></label>
+    {% if scantype.tag in ['T1'] %}
```

Comment:
Yeah. The metrics stuff is all such a mess that I left it alone for now, and just made a few minimal changes here and there to keep it from breaking other things. (And some changes I made before I realized how much work it would be to actually fix -all- of it, ha).